src/groups/bmq/bmqpi/bmqpi_dttracer.t.cpp

```diff
@@ -35,6 +35,17 @@ struct DTTracerTestImp : public bsls::ProtocolTestImp<bmqpi::DTTracer> {
         const bsl::shared_ptr<bmqpi::DTSpan>& parent,
         const bsl::string_view&               operation,
         const bmqpi::DTSpan::Baggage& baggage) const BSLS_KEYWORD_OVERRIDE;
+
+    virtual int serializeSpan(bsl::vector<unsigned char>*           out,
```

Comment:
```suggestion
    int serializeSpan(bsl::vector<unsigned char>*           out,
```

Can simplify, but also need to run formatter